va/caa_test.go

```diff
@@ -477,10 +476,6 @@ func TestCAALogging(t *testing.T) {
 		ChallengeType   core.AcmeChallenge
 		ExpectedLogline string
 	}{
-		{
-			Domain:          "reserved.com",
-			ExpectedLogline: "INFO: [AUDIT] Checked CAA records for reserved.com, [Present: true, Account ID: unknown, Challenge: unknown, Valid for issuance: false] Response=\"foo\"",
-		},
```

Comment:
Why was this test case removed?

Reply:
Because the only thing it's testing for is "do we print `unknown` for the Account ID and Challenge when they're not provided", and that's no longer a valid call. (This test function is specifically about the logging, not error conditions, so the test for erroring out when those aren't provided is in TestIsCAAValidParams below.)